service/src/main/java/uk/gov/hmcts/reform/fpl/config/DocmosisDocumentGenerationConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DocmosisDocumentGenerationConfiguration {
```

Comment:
Is this different from [DocmosisConfiguration](https://github.com/hmcts/fpl-ccd-configuration/blob/dev/spike-sdo/service/src/main/java/uk/gov/hmcts/reform/fpl/config/DocmosisConfiguration.java) class?

Reply:
exactly the same, removed